src/RoleObjects/MultiLineTextBoxData.js

```diff
@@ -283,7 +283,9 @@ export default class MultiLineTextBoxData extends AccessibilityObject {
    * @returns {String} string value for the text input field
    */
   get value() {
-    return this._reactProps.value;
+    const elementId = this._reactProps.id;
+    const element = document.getElementById(elementId);
+    return element.value;
```

Comment:
- this is a fix... `value` prop was never set (it was changed to store on the DOM element itself in a previous PR)